### PR TITLE
eloop per thread, better debugging

### PR DESCRIPTION
**Pull request by Lassulus**

### CODE REVIEW

File: src/Main.hs [hunk 1]
```diff
@@ -49,6 +49,6 @@ main = do
       bloom <- atomically $ newTVar bloom0
       eloop $ forConcurrently_
         (toList configBots)
-        (\(nick, bot) -> botThread bloom nick bot config)
+        (\(nick, bot) -> eloop $ botThread bloom nick bot config)
```
Comment (kmein) [inline src/Main.hs:52]: Do we then still need the outer eloop?

Comment (Lassulus) [reply src/Main.hs:52]: not sure, maybe stuff can break while parsing the json?
but it would be nonsense to try it again then I guess :D

Comment (kmein) [reply src/Main.hs:52]: Thats how it seems to me :laughing:

**Review by kmein — CHANGES_REQUESTED** [review]
Remove the outer `eloop` and we're good to go, I guess

**Comment by Lassulus** [comment]
done

**Pull request merged**